vnext/Mso/src/dispatchQueue/queueService.h

```diff
@@ -81,14 +81,16 @@ struct QueueLocalValueEntry {
 
 struct DispatchQueueStatic : Mso::UnknownObject<Mso::RefCountStrategy::NoRefCount, IDispatchQueueStatic> {
   static DispatchQueueStatic *Instance() noexcept;
-  static Mso::CntPtr<IDispatchQueueScheduler> MakeLooperScheduler() noexcept;
+  static Mso::CntPtr<IDispatchQueueScheduler> MakeLooperScheduler(
+      winrt::Microsoft::ReactNative::IReactNotificationService notificationService) noexcept;
```

Comment:
>IReactNotificationService [](http://example.com/codeflow?start=37&length=25)

Mso must not have a reference to React specific types. It is a generic  library similar Meta's Folly.
We must add Mso-specific abstraction implemented by ReactNative.

Reply:
I would propose to add a new struct DispatchQueueNotifications in this file. It must have three public Mso::Functor fields for the three new notifications. The functors are called if they are not empty (conversion to bool is true inside of 'if' statement).

Reply:
Sorry that I don't fully understand this part. Could you be a bit more specific? Maybe provide some code examples, they are usually very helpful to me.

Reply:
I can augment your PR with the approach that I would like the Mso code to changed if you don’t meet mind.

Reply:
Yes please, that would be highly appreciated. Thank you so much.

Reply:
I am traveling today. I will try to do it by the end of the week.

Reply:
@xidachenfb I had augmented the PR to avoid dependency of Mso on MS.RN.
The solution still needs to be tested.

Reply:
Just tested, and it is working, yay!